<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>731</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="SandboxCanvas" name="testLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>601</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gateButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>gate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="wireButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>wire</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>select</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="collapseButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>collapse</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SandboxCanvas</class>
   <extends>QLabel</extends>
   <header>sandboxcanvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
